pyramid_zipkin/tween.py

```diff
@@ -115,6 +116,7 @@ def _get_settings_from_request(request):
     zipkin_host = settings.get('zipkin.host')
     zipkin_port = settings.get('zipkin.port', request.server_port)
     firehose_handler = settings.get('zipkin.firehose_handler')
+    post_processor_callback = settings.get('zipkin.post_processor_callback')
```

Comment:
what's the reason behind the name? "processor" is confusing to me.

maybe something like "post_handler_hook"?

Reply:
post_handler_hook makes sense.